### PR TITLE
Release `v0.12.2-3`

### PR DESCRIPTION
**Pull request by AurevoirXavier**
> https://github.com/darwinia-network/darwinia-common/pull/1256

### CODE REVIEW

**Comment by HackFisher** [comment]
Please bump dep

**Comment by AurevoirXavier** [comment]
Waiting fork-off-substrate-test.

File: runtime/crab/src/migrations.rs [hunk 1]
```diff
@@ -36,24 +23,56 @@ impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
...
+
+	let now = System::block_number();
+
+	migration::storage_iter::<StakingLedger<AccountId, Balance, Balance, BlockNumber>>(
+		b"Staking", b"Ledger",
+	)
+	.for_each(
+		|(
+			_k,
+			StakingLedger {
```
Comment (HackFisher) [inline runtime/crab/src/migrations.rs:37]: Is this type Option?

**Review by HackFisher — APPROVED** [review]

File: runtime/crab/src/migrations.rs [hunk 2]
```diff
@@ -36,24 +23,56 @@ impl OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 }
 
 fn migrate() -> Weight {
-	darwinia_balances::migration::migrate::<Runtime, RingInstance>();
-	darwinia_balances::migration::migrate::<Runtime, KtonInstance>();
+	// --- darwinia-network ---
+	use darwinia_staking::{StakingLedger, STAKING_ID};
+
+	let now = System::block_number();
```
Comment (AurevoirXavier) [inline runtime/crab/src/migrations.rs:29]: Use a hardcode number to test with fork-off-substrate.
Otherwise `now` is 1 in the new chain.

**Pull request merged**